src/doc.c

```diff
@@ -2325,6 +2327,40 @@ void highlightText(Scope *sc, Dsymbol *s, OutBuffer *buf, size_t offset)
                 }
                 break;
 
+            case '`':
+                if (inBacktick)
+                {   inBacktick = 0;
```

Comment:
Yeah I know that argument is thrown around sometimes, but for this specific case both Andrei and I (and others) dislike the brace+code on the same line. :]